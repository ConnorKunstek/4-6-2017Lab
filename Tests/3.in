SSSSS
SDDDS
SDFDS
SDDDS
SSSSS
1 1
2 2
3 3
4 4
