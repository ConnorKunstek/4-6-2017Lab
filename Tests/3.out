No
Yes
Yes
Yes
Time: 0.00048 seconds

